tests/end2end/features/search.feature

```diff
@@ -238,3 +238,24 @@ Feature: Searching on a page
         Then the following tabs should be open:
             - data/search.html
             - data/hello.txt (active)
+
+    @qtwebkit_skip: Not supported in qtwebkit
```

Comment:
Oh, `:follow-selected` doesn't recurse into iframes in QtWebKit? 😆

Reply:
yup, that's what CI was telling me at least, I can't test qtwebkit right now, but I don't think we need to hold this back in the meantime...

Reply:
I added this to #3569.